packages/dashboard/src/components/FooterLine/About.tsx

```diff
@@ -0,0 +1,163 @@
+import { DiscordIcon, MaskBlueIcon, MaskGreyIcon, MaskTextIcon, MaskTextNightlyIcon } from '@masknet/icons'
+import { Avatar, IconButton, Link, makeStyles, Typography } from '@material-ui/core'
+import FacebookIcon from '@material-ui/icons/Facebook'
+import GitHubIcon from '@material-ui/icons/GitHub'
+import TelegramIcon from '@material-ui/icons/Telegram'
+import TwitterIcon from '@material-ui/icons/Twitter'
+import { useDashboardI18N } from '../../locales'
+import { experimentalStyled as styled } from '@material-ui/core/styles'
+import { Version } from './Version'
+import { getMaskColor } from '@masknet/theme'
+import links from './links.json'
+import { LOCAL_IMAGE } from '../../assets'
+
+const useStyles = makeStyles((theme) => ({
+    wrapper: {
+        width: 580,
+        minHeight: 660,
+        lineHeight: 1.75,
+    },
+    header: {
+        height: 300,
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        background: `url(${LOCAL_IMAGE.ABOUT_DIALOG_BACKGROUND}) no-repeat center / cover`,
+    },
+    maskface: {
+        width: 120,
+        height: 120,
+        marginTop: 75,
+    },
+    version: {
+        color: '#FFF',
+        fontSize: 12,
+        marginBottom: 20,
+    },
+    main: {
+        fontSize: 16,
+        textAlign: 'center',
+        margin: '24px 68px',
+
+        '& > p': {
+            marginBottom: '36px',
+        },
+    },
+    getInTouch: {
+        fontSize: 16,
+        fontWeight: 'bold',
+        marginTop: 20,
+        marginBottom: 28,
+    },
+    icon: {
+        color: theme.palette.text.primary,
+    },
+    close: {
+        color: '#FFF',
+    },
+    brands: {
+        marginTop: theme.spacing(1),
+        '& > *': {
+            margin: theme.spacing(0, 1),
+            cursor: 'pointer',
+        },
+    },
+    footer: {
+        borderTop: `1px solid ${theme.palette.divider}`,
+        color: getMaskColor(theme).iconLight,
+        fontSize: '0.77rem',
+        margin: theme.spacing(0, 2),
+        padding: theme.spacing(2, 2, 3, 6),
+    },
+    link: {
+        color: getMaskColor(theme).iconLight,
+    },
+}))
+
+const IconContainer = styled('div')(
+    ({ theme }) => `
+    width: 100%;
+    height: 100%;
+
+    & > svg {
+        width: 100%;
+        height: 100%;
+    }
+`,
+)
+const TextIconContainer = styled('div')(
+    ({ theme }) => `
+    padding: ${theme.spacing(1)} 0;
+    width: 100px;
+
+    & > svg {
+        width: 100%;
+        height: 100%;
+    }
+`,
+)
+
+const brands: Record<string, React.ReactNode> = {
+    'https://www.facebook.com/masknetwork': <FacebookIcon />,
+    'https://twitter.com/realmaskbook': <TwitterIcon />,
+    'https://github.com/DimensionDev/Maskbook': <GitHubIcon />,
+    'https://t.me/maskbook_group': <TelegramIcon />,
+    'https://discord.gg/4SVXvj7': <DiscordIcon />,
+}
```

Comment:
@yisiliu request provide shortlink service.

```
r2d2.to/facebook
r2d2.to/twitter
r2d2.to/github
r2d2.to/telegram
r2d2.to/discord
```
etc

Reply:
This could be useful.